Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -105,6 +105,16 @@ public class BellerophonManager: NSObject {
             }
         }
     }
+    
+    public func dismissKillSwitchIfNeeded() {
```

Comment:
The name isn't very explicit, maybe keep this function internal, and create a new public function with a more descriptive name that calls this one?

Reply:
@dominicancrum any updates on this PR?

Reply:
With the recent changes to the kill switch  I think this can actually be closed @ThibaultKlein